docs/example.rst

```diff
@@ -19,39 +16,33 @@ Example
       -[channel] *120*456#
 
 
-   Favourite Colour
-   ~~~~~~~~~~~~~~~~
+   ### Favourite Colour
```

Comment:
From user testing, people struggled to differentiate between the different levels (dialogue, sequence, block) with underlining. `###`s solve this, since one can more easily differentiate between the different levels.